uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/CheckoutActivity.kt

```diff
@@ -133,8 +133,8 @@ class CheckoutActivity : BaseActivity(), WebViewActions {
          * destination and date of the desired trip. This will only use the details available inside
          * the BookingInfo object.
          */
-        fun bookingInfo(bookingInfo: BookingInfo): Builder {
-            extrasBundle.putParcelable(BOOKING_CHECKOUT_STATUS_KEY, bookingInfo)
+        fun bookingInfo(journeyDetails: JourneyDetails): Builder {
```

Comment:
You can rename the method as well.

Reply:
Also the comment before the method

Reply:
May be a solution to add @Deprecated to the bookingInfo method and add another one called journeyDetails not to impact current DP implementations?

Reply:
In general it's a good approach, but in this case we'll have breaking changes with this component anyway. And afaik nobody is implementing the quote list as a standalone component at this time, so no DP should be impacted by this. The only things we'd need to revise are the sample apps.
@mohadian could you please confirm or correct me if I'm wrong?